src/VisualStudio/Core/Impl/Options/AbstractOptionPageControl.cs

```diff
@@ -67,38 +72,37 @@ protected void BindToOption(CheckBox checkbox, Option<bool> optionKey)
             _bindingExpressions.Add(bindingExpression);
         }
 
-        protected void BindToOption(CheckBox checkbox, Option<int> optionKey)
+        protected void BindToOption(CheckBox checkbox, PerLanguageOption<bool> optionKey, string languageName)
```

Comment:
Why are CheckBox-related things changing?

Reply:
With the "Use Enhanced Colors for C# and VB" checkbox removed there is no need to keep around the converter for "tri-state" integer driven checkboxes.